Checklist for PR (#1410)
CDCgov/prime-simplereport: .github/pull_request_template.md

- complaints about how bad the code is

## Screenshots / Demos

## Checklist for Author and Reviewer

> 

Reviewer: ```suggestion
### UI
- [ ] Any changes to the UI/UX have been approved by design 
```

### Testing
- [ ] Includes a summary of what a code reviewer should verify

> - [ ] Changes are well-tested and code coverage remains the same or increases

Reviewer: ```suggestion
```

I think this is effectively enforced by by sonar

> ### Breaking Changes

Reviewer: ```suggestion
### Changes are Backwards Compatible
```

> - [ ] This PR does not introduce any breaking changes

Reviewer: ```suggestion
```

I would remove this. I think the next two check boxes effectively get at the know issues

- [ ] Database changes are submitted as a separate PR
- [ ] GraphQL schema changes are backward compatible with older version of the front-end

### Security

> - [ ] This PR does not introduce changes that require a security review (encryption changes, etc)

Reviewer: ```suggestion
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
```

- complaints about how bad the code is

## Screenshots / Demos

## Checklist for Author and Reviewer

### UI

> - [ ] Any changes to the UI/UX have been approved by design 

Reviewer: ```suggestion
- [ ] Any changes to the UI/UX have been approved by design 
- [ ] Any new or edited error messages have been approved by design 
```
